01_prepare_host.sh

```diff
@@ -55,7 +55,7 @@ podman)
 esac
 
 # Pull images we'll need
-for IMAGE_VAR in IRONIC_IMAGE IPA_DOWNLOADER_IMAGE VBMC_IMAGE SUSHY_TOOLS_IMAGE; do
+for IMAGE_VAR in IRONIC_IMAGE IPA_DOWNLOADER_IMAGE VBMC_IMAGE SUSHY_TOOLS_IMAGE DOCKER_REGISTRY_IMAGE; do
```

Comment:
We need to add the logic from dev-scripts `01_install_requirements.sh` to start the docker-distribution container, as this only pulls it I think?

Reply:
Oh nvm I see it's below...